scripts/src/internal/services.ts

```diff
@@ -169,16 +169,21 @@ export async function paymentComplete(payment: CompletedPayment, logger: LoggerI
 	}
```

Comment:
at line 160 there are comments regards p2p I guess there are already done by this PR, so please delete if so